pkgs/top-level/python-packages.nix

```
@@ -6897,6 +6897,8 @@ in {
 
   rig = callPackage ../development/python-modules/rig { };
 
+  ring_doorbell = callPackage ../development/python-modules/ring_doorbell { };
```

```suggestion
  ring-doorbell = callPackage ../development/python-modules/ring_doorbell { };
```
https://nixos.org/manual/nixpkgs/stable/#contributing

I think this also applies to `pname`.

---

I've changed this, thanks.  I presume it should also apply to the directory name (which I changed too)?  I had to override the `pname` to use underscores for fetchPyPI.